autopilot/autopilot.go

```diff
@@ -211,6 +199,28 @@ func (ap *Autopilot) Run() error {
 			ctx, span := tracing.Tracer.Start(context.Background(), "Autopilot Iteration")
 			defer span.End()
 
+			// initiate a host scan - no need to be synced or configured for scanning
+			ap.s.tryUpdateTimeout()
+			ap.s.tryPerformHostScan(ctx, w, forceScan)
+
+			// block until the autopilot is configured
+			if !ap.blockUntilConfigured(ap.ticker.C) {
```

Comment:
Could we (or should we) add `ap.isConfigured()` and `ap.isSynced()` checks here to escape early once the initial run went through both `blockX` functions successfully. If not we should remove `sync` and `configured` from the autopilot because they are not being used. I think maybe they were once used and not cleaned up and/or added and simply not used. Probably my fault but we can address it here.

Reply:
Ok I removed them. The reason why I didn't perform a shortcut to begin with was to account for the unlikely case where a user manually deletes the config, resyncs the chain without stopping the autopilot etc.